### PR TITLE
chore: add tests for customerCount method

### PR DESCRIPTION
**Pull request by ansmonjol**
## Context

[A recent bugfix](https://github.com/getlago/lago-api/pull/620/files) did correct the value returned in some methods

## Description

This PR adds tests to assert the value.

### CODE REVIEW

**Review by jdenquin — APPROVED** [review]

File: spec/graphql/resolvers/plan_resolver_spec.rb [hunk 1]
```diff
@@ -33,6 +38,8 @@
 
     aggregate_failures do
       expect(plan_response['id']).to eq(plan.id)
+      expect(plan_response['customerCount']).to eq(1)
+
```
Comment (jdenquin) [inline spec/graphql/resolvers/plan_resolver_spec.rb:42]: Layout/EmptyLinesAroundBlockBody: Extra empty line detected at block body end. (https://rubystyle.guide#empty-lines-around-bodies)

Comment (ansmonjol) [reply spec/graphql/resolvers/plan_resolver_spec.rb]: Fixed

File: spec/graphql/resolvers/add_on_resolver_spec.rb [hunk 2]
```diff
@@ -29,6 +29,10 @@
   end
 
   it 'returns a single add-on' do
+    3.times do
```
Comment (vincent-pochet) [inline spec/graphql/resolvers/add_on_resolver_spec.rb:32]: You should put this into a `before` block

File: spec/graphql/resolvers/plans_resolver_spec.rb [hunk 3]
```diff
@@ -16,10 +16,15 @@
 
   let(:membership) { create(:membership) }
   let(:organization) { membership.organization }
+  let(:customer) { create(:customer, organization: organization) }
 
   it 'returns a list of plans' do
     plan = create(:plan, organization: organization)
 
+    2.times do
```
Comment (vincent-pochet) [inline spec/graphql/resolvers/plans_resolver_spec.rb:24]: and `before` :p

**Review by vincent-pochet — APPROVED** [review]

File: spec/graphql/resolvers/add_on_resolver_spec.rb (hunk 2 above)
Comment (ansmonjol) [reply spec/graphql/resolvers/add_on_resolver_spec.rb:30]: All done ! Thanks for your comments

**Pull request merged**